torch/_tensor_docs.py

```diff
@@ -5360,10 +5360,16 @@ def callable(a, b) -> number
 add_docstr_all(
     "to_dense",
     r"""
-to_dense() -> Tensor
+to_dense(dtype=None, *, masked_grad=None) -> Tensor
```

Comment:
nit: Did you mean `masked_grad=True`?

Reply:
I guess in the docs it is ok to use `masked_grad=True` (I'll fix it). Otherwise, in the implementations, we use `masked_grad=None` to make the deprecation and/or changing of the default processes easier.

Reply:
Fixed.